lib/utils/theme.dart

```diff
@@ -104,48 +105,62 @@ ThemeData _generateTheme(Brightness brightness) {
 
   const iconTheme = IconThemeData(size: 32);
   const labelTextStyle = StyleText(size: 12, weight: 600);
+  final inputLabelStyle = WidgetStateTextStyle.resolveWith(
+    (states) => TextStyle(color: states.isFocused ? green : textColor),
+  );
 
-  MaterialStateProperty<T> selected<T>(T selected, T unselected) =>
-      MaterialStateProperty.resolveWith((states) => states.isSelected ? selected : unselected);
+  WidgetStateProperty<T> selected<T>(T selected, T unselected) =>
+      WidgetStateProperty.resolveWith((states) => states.isSelected ? selected : unselected);
 
-  MaterialStateProperty<T> tealWhenSelected<T>(T Function({Color color}) copyWith, bool isLight) {
+  WidgetStateProperty<T> tealWhenSelected<T>(T Function({Color color}) copyWith, bool isLight) {
     return selected(
       copyWith(color: ThcColors.teal),
-      copyWith(color: paleColor.withOpacity(0.33)),
+      copyWith(color: paleColor.withOpacity(1 / 3)),
     );
   }
 
   return ThemeData(
-    colorScheme: ColorScheme(
+    colorScheme: ColorScheme.fromSeed(
+      seedColor: ThcColors.darkBlue,
       brightness: isLight ? Brightness.light : Brightness.dark,
-      primary: isLight ? ThcColors.green : ThcColors.zaHando,
-      primaryContainer: isLight ? ThcColors.dullGreen38 : ThcColors.dullGreen50,
+      primary: green,
       onPrimary: isLight ? ThcColors.dullerGreen : Colors.black87,
       inversePrimary: ThcColors.darkGreen,
-      secondary: ThcColors.teal,
-      onSecondary: Colors.white,
-      tertiary: isLight ? ThcColors.darkMagenta : ThcColors.tan,
+      primaryContainer: isLight ? ThcColors.dullGreen38 : ThcColors.dullGreen50,
+      secondary: ThcColors.dullBlue,
+      onSecondary: paleColor,
+      tertiary: ThcColors.teal,
       onTertiary: isLight ? ThcColors.tan : ThcColors.darkMagenta,
-      error: Colors.red.withOpacity(0.75),
-      onError: Colors.white,
-      errorContainer: Colors.red.withOpacity(0.33),
-      onErrorContainer: Colors.red,
-      background: isLight ? _lightBackground : ThcColors.darkestBlue,
-      onBackground: textColor,
-      surface: isLight ? ThcColors.tan : ThcColors.dullBlue,
+      surface: backgroundColor,
       onSurface: textColor,
       inverseSurface: isLight ? ThcColors.darkBlue : ThcColors.paleAzure,
-      onInverseSurface: isLight ? Colors.white : ThcColors.darkBlue,
-      surfaceVariant: ThcColors.dullBlue,
-      onSurfaceVariant: paleColor,
+      onInverseSurface: isLight ? Colors.white : ThcColors.darkestBlue,
       outline: slightContrast,
       outlineVariant: contrast25,
     ),
```

Comment:
A lot of changes here, but it boils down to 2 things:
- `ColorScheme.fromSeed()` is the ✨newest✨ color scheme constructor
- Names have been updated to match the [Material 3 website](https://m3.material.io/styles/color/system/how-the-system-works#7e1088b5-2667-4bf1-b731-cb7eb290babe)